test/statsCases/chunks/expected.txt

```diff
@@ -6,18 +6,44 @@ Time: Xms
 2.bundle.js  101 bytes       2  [emitted]  
 3.bundle.js  180 bytes       3  [emitted]  
 chunk    {0} bundle.js (main) 73 bytes [rendered]
+    > main [0] (webpack)/test/statsCases/chunks/index.js 
```

Comment:
The default did not change at all.  What I noticed when writing the tests was that the `"should print correct stats for ..."` test [did not actually use the stats config object](https://github.com/webpack/webpack/blob/master/test/Stats.test.js#L47-L49) that was included with several tests.

You can test this by changing the values of the `stats` object in any of the [webpack.config.js](https://github.com/webpack/webpack/blob/master/test/statsCases/chunks/webpack.config.js) test files;  on webpack/master, the tests will (wrongly) never break.

The `simple` and `simple-more-info` tests gave the illusion that `stats` config mattered, but it was only because of the compiler `profile` option, that is picked up by Stats via `module.profile`.